### PR TITLE
Quantified Elements

### PR DESCRIPTION
**Pull request by klh6157**
Web components done:
1. Tagging questions
2. Self-check
3. Inline-audio
4. Flash card /lib flash card answer box
5. Video Player
6. Haxor slevin
7. Training theme
8. Hal 9000 /lib Hal 9000 ui
9. Hax-body /lib hax-map
10. Hax-body /lib hax-tray

### CODE REVIEW

File: elements/inline-audio/src/inline-audio.js [hunk 1]
```diff
@@ -29,7 +31,7 @@ Custom property | Description | Default
  * @element inline-audio
  * 
  */
-class InlineAudio extends I18NMixin(SimpleColors) {
+class InlineAudio extends I18NMixin(SimpleColors, DDD) {
```
Comment (btopro) [inline elements/inline-audio/src/inline-audio.js:34]: `I18NMixin(DDD)` and not `SimpleColors,DDD`

Make sure to fix this in the other elements where it's happened

File: elements/app-hax/lib/v1/app-hax-search-results.js [hunk 2]
```diff
@@ -1,196 +0,0 @@
-/* eslint-disable no-return-assign */
```
Comment (btopro) [inline elements/app-hax/lib/v1/app-hax-search-results.js:1]: uhhh.. this file should not be wiped

File: elements/flash-card/lib/flash-card-answer-box.js [hunk 3]
```diff
@@ -4,8 +4,9 @@ import { SimpleColors } from "@haxtheweb/simple-colors/simple-colors.js";
 import "@haxtheweb/simple-icon/lib/simple-icons.js";
 import "@haxtheweb/simple-icon/lib/simple-icon-lite.js";
 import "@haxtheweb/simple-icon/lib/simple-icon-button-lite.js";
+import { DDD } from "@haxtheweb/d-d-d";
 
-export class FlashCardAnswerBox extends I18NMixin(SimpleColors) {
+export class FlashCardAnswerBox extends I18NMixin(SimpleColors, DDD) {
```
Comment (btopro) [inline elements/flash-card/lib/flash-card-answer-box.js:9]: fix

File: elements/simple-icon/src/simple-icon.js [hunk 4]
```diff
@@ -21,7 +22,7 @@ import { SimpleIconBehaviors } from "./lib/simple-icon-lite.js";
  * @demo demo/iconset.html Iconset Demo
  * @element simple-icon
  */
-class SimpleIcon extends SimpleIconBehaviors(SimpleColors) {
+class SimpleIcon extends SimpleIconBehaviors(SimpleColors, DDD) {
```
Comment (btopro) [inline elements/simple-icon/src/simple-icon.js:25]: fix

File: elements/tagging-question/src/tagging-question.js [hunk 5]
```diff
@@ -65,8 +66,8 @@ class TaggingQuestion extends QuestionElement {
           border-radius: var(--ddd-radius-sm);
           box-sizing: border-box;
           background-color: light-dark(
-            var(--simple-colors-default-theme-grey-2),
-            var(--simple-colors-default-theme-grey-10)
+            var(--ddd-theme-default-coalyGray),
```
Comment (btopro) [inline elements/tagging-question/src/tagging-question.js:69]: no need for light-dark if this is going to be the same background color regaudless

File: elements/tagging-question/src/tagging-question.js [hunk 6]
```diff
@@ -98,16 +99,16 @@ class TaggingQuestion extends QuestionElement {
           border-style: dashed;
           border-color: black;
           background-color: light-dark(
-            var(--simple-colors-default-theme-grey-3),
-            var(--simple-colors-default-theme-grey-9)
+            var(--ddd-theme-default-disable),
```
Comment (btopro) [inline elements/tagging-question/src/tagging-question.js:102]: same as prev comment

**Review by btopro — CHANGES_REQUESTED** [review]
bunch of things to clean up

File: elements/tagging-question/tagging-question.js [hunk 7]
```diff
@@ -65,7 +65,7 @@ class TaggingQuestion extends QuestionElement(DDD) {
           border: var(--ddd-border-sm);
           border-radius: var(--ddd-radius-sm);
           box-sizing: border-box;
-          background-color: light-dark(
+          background-color: (
```
Comment (btopro) [inline elements/tagging-question/tagging-question.js:68]: if it's not changing between light and dark in this way then it can just be

`background-color: var(--ddd-theme-default-coalyGray);`

File: elements/data-viz/src/data-viz.js [hunk 8]
```diff
@@ -4,7 +4,6 @@
  */
 import { LitElement, html, css } from "lit";
 import "@haxtheweb/chartist-render/chartist-render.js";
-import "@haxtheweb/es-global-bridge/es-global-bridge.js";
```
Comment (btopro) [inline elements/data-viz/src/data-viz.js]: why did you delete this line?

Comment (klh6157) [reply elements/data-viz/src/data-viz.js]: This was done by me, I added it and then deleted it. I misread the error, It was related to something in chartist-render.

File: elements/d-d-d/package.json [hunk 9]
```diff
@@ -47,6 +47,9 @@
     "@haxtheweb/hax-iconset": "^9.0.6",
     "@haxtheweb/simple-colors": "^9.0.6",
     "@haxtheweb/simple-icon": "^9.0.6",
+    "@haxtheweb/page-section": "^9.0.6",
+    "@haxtheweb/haxcms-elements": "^9.0.6",
+    "@haxtheweb/simple-modal": "^9.0.6",
```
Comment (btopro) [inline elements/d-d-d/package.json:52]: @klh6157 make these 3 dependencies show up under devDependencies. This is that weird package where we don't want ppl needing to install these just for the demo to work when it would invoke a ridiculous downstream series of dependencies (that will never get used) just so that we can demo DDD. I will make a new issue about splitting these out because of how absurd it is :)

Comment (klh6157) [reply elements/d-d-d/package.json]: Yea, on Friday I will fix that. Which other elements should fall into this category? or do you think I should still put the others into dependency for now?

Comment (klh6157) [reply elements/d-d-d/package.json]: oh ok, thats good to know

**Comment by btopro** [comment]
open a new PR / keep submitting new things after you rebase your repo to ensure it's up to date tomorrow when you get in.
